server/auth/authenticate.js

```diff
@@ -0,0 +1,42 @@
+//Adding jwt token 
+const jwt = require('jsonwebtoken')
+
+const jwtKey =
+process.env.JWT_SECRET || "supercalifragilisticexpialidocious"
+
+
+// quickly see what this file exports
+module.exports = {
+    authenticate,
+    generateToken
+  };
+
+// To authenticate the user  
+function authenticate (req, res, next)  {
+    const token = req.get("Authorization")
+
+    if(token) {
+        jwt.verfy(token, jwtKey, (err, decoded) => {
```

Comment:
`verfy` is spelled incorrectly, should be `jwt.verify`.

Reply:
Yea not sure how this happened  either I will need to revise it although it is not breaking an code. As it is not linked to anything